worker/tsa_config.go

```diff
@@ -9,7 +9,7 @@ import (
 type TSAConfig struct {
 	Hosts            []string            `long:"host" default:"127.0.0.1:2222" description:"TSA host to forward the worker through. Can be specified multiple times."`
 	PublicKey        flag.AuthorizedKeys `long:"public-key" description:"File containing a public key to expect from the TSA."`
-	WorkerPrivateKey *flag.PrivateKey    `long:"worker-private-key" description:"File containing the private key to use when authenticating to the TSA."`
+	WorkerPrivateKey *flag.PrivateKey    `long:"worker-private-key" required:"true" description:"File containing the private key to use when authenticating to the TSA."`
```

Comment:
🤔 This is kinda just snuck in here but I guess it's fine. The intent was to make it easier to see if the worker can even run on its own before worrying about to registering it with the cluster. I'm not sure if anyone knows about it though.

Reply:
@vito yeah, we saw it as part of the refactor of `cmd/concourse/worker.go` to separate out the volume and container sweepers, but I guess we should have made it more explicit. Do you want us to make it a separate commit?

Reply:
That's OK, let's just see if anyone complains about this being required now. It might be more obvious this way.